docs/3.x.x/guides/databases.md

```diff
@@ -0,0 +1,296 @@
+# Databases
+
+Strapi gives you the option to choose the most appropriate database for your project. It currently supports **PostgreSQL**, **MongoDB**, **SQLite**, **MySQL** and
+**MariaDB**. The following documentation covers how to install these databases locally (for development purposes) and on various hosted or cloud server solutions (for staging or production purposes).
+
+(Deploying **Strapi** itself is covered in the [Deployment Guide](/3.x.x/guides/deployment.html).)
+
+**Table of contents:**
+
+- [SQLite Installation](#sqlite-installation)
+- [MongoDB Installation](#mongodb-installation)
+
+
+---
+
+## SQLite Installation
+
+SQLite is the default ([Quick Start](/3.x.x/getting-started/quick-start.html)) and recommended database to quickly deploy an app locally.
+
+### Install SQLite locally
+
+Simply use the `strapi new my-project --quickstart` command.
+
+```bash
+strapi new my-project --quickstart
+```
+
+This will create a new project and launch it in the browser. (The [Quick Start Guide](/3.x.x/getting-started/quick-start.html) is a complete step-by-step tutorial.)
+
+## MongoDB Installation
+
+### Install MongoDB locally
+
+In order to use Strapi with [MongoDB](https://www.mongodb.com/) in production, you must have and use MongoDB on your local development environment. These instructions show how to install a Strapi project locally with a MongoDB database.
```

Comment:
These instructions show how to start using Strapi with a MongoDB database.

Reply:
done